frontend/src/main/java/org/abs_models/Absc.java

```diff
@@ -195,6 +195,29 @@ public LocationType convert(String value) throws Exception {
     // LocationTypeInferrerExtension.defaultType
     public LocationType defaultLocationType = LocationType.INFER;
 
+    //mtvl
```

Comment:
These options have been moved to the class `org.abs_models.frontend.typechecker.CheckSPLCommand`; check the output of `absc checkspl --help`.

Reply:
Yup I had notice this that the analysis operation had been move to absc checkspl. I had moved to implementing it in checkspl. This pull request was a mistake few day back and I will close this full request now.